docs/release-notes/8.0.asciidoc

```diff
@@ -46,6 +46,23 @@ TIP: You can search for the rule by entering the rule name in the Rule table's s
 . Save the changes.
 . Activate the rule.
 
+*The import process fails for rules with exception comments*
+
+Comments on rule exceptions will cause the import process to fail as the following system-generated attributes cannot be validated for exception comments ({pull}124742[#124742]):
+
+ * `created_at`
+ * `created_by`
+ * `updated_at`
+ * `updated_by`
+
+To complete the rule import process successfully, update the exported `.ndjson` file and re-import it:
+
+. Search the exported `.ndjson` file for exceptions with comments. Exception comments are stored within the `exceptionItem` object (`exceptionItem.comments`).
```

Comment:
When they search they should just search for `comments` as rules don't have that field so it would just point them to the exception item comments. 
```suggestion
. Search the exported `.ndjson` file for exceptions with comments. Exception comments are stored within the `exceptionItem` object under the `comments` field.
```